.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "dotenv.enableAutocloaking": false
+}
```

Comment:
You should add this folder to .gitignore

[Check this StackOverflow](https://stackoverflow.com/questions/6313126/how-do-i-remove-a-directory-from-a-git-repository)

Reply:
done